wagtail/admin/templates/wagtailadmin/pages/listing/_button_with_dropdown.html

```diff
@@ -4,14 +4,16 @@
         <div data-dropdown-toggle class="o-icon c-dropdown__toggle  [ icon icon-arrow-down ]"></div>
     </a>
     <div class="t-dark">
-        <ul role="menu" class="c-dropdown__menu u-toggle  u-arrow u-arrow--tl u-background">
-        {% for button in buttons %}
-            <li class="c-dropdown__item ">
-                <a href="{{ button.url }}" title="{{ button.attrs.title }}" class="u-link is-live {{ button.classes|join:' ' }}">
-                    {{ button.label }}
-                </a>
-            </li>
-        {% endfor %}
-        </ul>
+        <nav aria-label="More options">
```

Comment:
This is a reusable component for which we don’t know how and how often it will be used, so I don’t think this should be set up as a `nav` landmark.

Reply:
I will remove.